build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+buildscript {
+    val kotlin_version by extra("1.3.71")
+    repositories {
+        jcenter()
+        google()
+        maven { url = uri("http://oss.sonatype.org/content/repositories/snapshots/") }
+    }
+    dependencies {
+        classpath("com.vanniktech:gradle-maven-publish-plugin:0.9.0")
+        classpath("com.android.tools.build:gradle:3.6.1")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
+        classpath("org.jetbrains.dokka:dokka-gradle-plugin:0.10.0")
+    }
+}
+
+plugins {
+    id("io.gitlab.arturbosch.detekt").version("1.7.2")
+    id("com.github.ben-manes.versions").version("0.28.0")
+}
+
+dependencies {
+    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.7.2")
```

Comment:
This includes ktlint formatting as part of detekt, if you were to write a custom detekt rule it would be included here too I believe